ricecooker/classes/files.py

```diff
@@ -1030,15 +1030,15 @@ class _ExerciseGraphieFile(DownloadFile):
 
     def __init__(self, path, **kwargs):
         self.original_filename = (
-            path.split("/")[-1].split(os.path.sep)[-1].split(".")[0]
+            path.split(os.path.sep)[-1].split(".")[0]
         )
         super(_ExerciseGraphieFile, self).__init__(path, **kwargs)
 
     def get_preset(self):
         return self.preset or format_presets.EXERCISE_GRAPHIE
 
     def get_replacement_str(self):
-        return self.path.split("/")[-1].split(".")[0] or self.path
+        return self.path.split('/')[-1].split(".")[0] or self.path
```

Comment:
This doesn't seem to have actually changed? I think maybe we could just return `self.original_filename` here, to avoid reduplicating the logic?

Reply:
Nothing is changed here